tests/containers/container_diff.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Print and save diffs between two cotaniners using container-diff tool
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base 'consoletest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+use containers::container_images;
+use suse_container_urls 'get_suse_container_urls';
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my ($image_names, $stable_names) = get_suse_container_urls();
+
+    for my $i (0 .. $#$image_names) {
+        perform_container_diff($image_names->[$i], $stable_names->[$i]) if (check_var("ARCH", "x86_64"));
```

Comment:
Is this meant to be the only part where `perform_container_diff` is called? If so, this could be inlined and removed from `lib/` again.

Reply:
Yes, currently we call `perform_container_diff()` only from `container_diff.pm` test module. But as we may call it multiple times here because of the for loop I would prefer to keep this function. I moved the function to the test module file. Is it OK for you?

Reply:
Actually, inlining it properly would make the code much shorter and also get rid of unnecessary `which container-diff` calls. You can also drop the argument validation and such.

Reply:
OK, you're right. Now it's inlined.